src/App.js

```diff
@@ -141,18 +141,15 @@ function App() {
   const [guess, setGuess] = useState("");
   const [themeMode, setThemeMode] = useState(browserThemeMode());
   const [giveup,setGiveup] = useState(false);
+  const [foundMatches, setFoundMatches] = useState(0);
```

Comment:
I don't see `foundMatches` being used anywhere, other than to calculate the next foundMatches. Is this needed?

Reply:
eliminated 'foundMatches'.